### PR TITLE
Update the lapcap_data_template_master seed data to list Northern Ireland by its full name rather than its initials

### PR DESCRIPTION
**Pull request by tim-powell-eviden**
Update the lapcap_data_template_master seed data to list Northern Ireland by its full name rather than its initials

### CODE REVIEW

**Review by udaydefra — CHANGES_REQUESTED** [review]
MigrationSql changes missing

File: obj/EPR.Calculator.API.Data.EntityFrameworkCore.targets
Comment (udaydefra) [inline obj/EPR.Calculator.API.Data.EntityFrameworkCore.targets:1]: Why is this changed?

Comment (tim-powell-eviden) [reply obj/EPR.Calculator.API.Data.EntityFrameworkCore.targets:1]: No idea, Visual Studio did it automatically.

Comment (tim-powell-eviden) [reply obj/EPR.Calculator.API.Data.EntityFrameworkCore.targets:1]: I've removed the file.

**Review by udaydefra — APPROVED** [review]
LGTM

**Review by johnbenjamin-defra — APPROVED** [review]
